spec/tapioca/dsl/compilers/active_record_associations_spec.rb

```diff
@@ -225,6 +225,8 @@ def picture_ids; end
                         sig { params(ids: T::Array[T.untyped]).returns(T::Array[T.untyped]) }
                         def picture_ids=(ids); end
 
+                        # This method is created by ActiveRecord on the `Employee` class because it declared `has_many :pictures`.
```

Comment:
Can we add a test that includes YARD documentation? I want to make sure that the guides link is immediately visible when hovering without having to scroll.

Reply:
Oh, sorry, these are DSLs not gem RBIs. There's no YARD docs for those, so this won't be an issue indeed.

The control we'd have over it would be placing the link comments first, so that they are immediately visible when hovering.

Reply:
No, no. Indeed I confused them with gem RBIs, which may have YARD docs attached to them. But DSLs should not have any other comments as far as I remember.